docs/ReleaseNotes.md

```diff
@@ -12,7 +12,7 @@ The included licenses apply to the following files:
 
 | License file | Applies to |
 |---|---|
-|LICENSE-MS.txt     |dxil.dll (if included in package)|
```

Comment:
Since it's now built out of this repo, doesn't that mean that it should have the same license as any of the other executables?  Is the source code under a different license?

Reply:
Updated.